probe/endpoint/procspy/proc_linux.go

```diff
@@ -22,8 +22,19 @@ var (
 	procRoot               = "/proc"
 	namespaceKey           = []string{"procspy", "namespaces"}
 	netNamespacePathSuffix = ""
+	ipv6IsSupported        = tcp6FileExists()
 )
 
+func tcp6FileExists() bool {
+	filename := filepath.Join(procRoot, "self/net/tcp6")
+	f, err := fs.Open(filename)
+	if err != nil {
```

Comment:
Ideally we should be checking this is ENOENT, and log an error otherwise. I can imagine it being super confusing if some other crazy circumstance meant a different error and we had no clue why ipv6 conns weren't showing up.